libraries/standard/http/include/http_client.h

```diff
@@ -178,7 +187,7 @@ typedef enum HTTPStatus
  * @brief Represents header data that will be sent in an HTTP request.
  *
  * The memory for the header data buffer is supplied by the user. Information in
- * the buffer will be filled by calling #HTTPClient_InitializeHeaders.
+ * the buffer will be filled by calling #HTTPClient_InitializeRequestHeaders.
```

Comment:
Can you also add "and #HTTPClient_AdddHeader."

Reply:
Sure